### PR TITLE
Mo 173 display employees and projects in custom payroll

### PR DESCRIPTION
**Pull request by AndriiK139MCAF**

### CODE REVIEW

File: mo_hr_timesheets/wizard/payroll_reports.py [hunk 1]
```diff
@@ -15,97 +15,117 @@ class HrLeaveType(models.Model):
...
+                departments = self.env['hr.department'].search([('id', 'child_of', self.department_id.id)])
+                employees = self.env['hr.employee'].search([('department_id', 'in', departments.ids)])
+            else:
+                employees = self.employee_ids
 
-            calendar = employee.resource_calendar_id or self.env['resource.calendar'].search([], limit=1)
-            hours_per_day = calendar.hours_per_day
+            row = 2
+            for employee in employees:
+                date_from = fields.Date.from_string(self.date_from)
```
Comment (MORPSmcaf) [inline mo_hr_timesheets/wizard/payroll_reports.py:66]: move before loop

File: mo_hr_timesheets/wizard/payroll_reports.py [hunk 2]
```diff
@@ -15,97 +15,117 @@ class HrLeaveType(models.Model):
...
+                date_to = fields.Date.from_string(self.date_to)
```
Comment (MORPSmcaf) [inline mo_hr_timesheets/wizard/payroll_reports.py:67]: move before loop

Comment (AndriiK139MCAF) [reply mo_hr_timesheets/wizard/payroll_reports.py]: done

File: mo_hr_timesheets/wizard/payroll_reports.py (hunk 1 above)
Comment (AndriiK139MCAF) [reply mo_hr_timesheets/wizard/payroll_reports.py]: done

**Review by MORPSmcaf — APPROVED** [review]

**Review by MORPSmcaf — APPROVED** [review]

**Pull request merged**